src/app/issues-viewer/card-view/card-view.component.html

```diff
@@ -18,12 +18,14 @@
     </mat-card>
   </div>
 
-  <div *ngFor="let issue of this.issues$ | async; index as i">
-    <app-issue-pr-card [issue]="issue" [dropdownFilter]="issues.dropdownFilter"></app-issue-pr-card>
-  </div>
+  <mat-card-content style="position: relative;">
+    <div *ngFor="let issue of this.issues$ | async; index as i" class="scrollable-container" style="position: relative; z-index: -1;">
```

Comment:
Is this class used anywhere?

Reply:
I think I left it in there while I was experimenting, I'll remove it, sorry about that

Reply:
I think I left this here while experimenting, I'll rename it to something more appropriate and move the styles to the css